app-backend/database/src/main/scala/com/azavea/rf/database/tables/Buckets.scala

```diff
@@ -118,6 +118,25 @@ object Buckets extends TableQuery(tag => new Buckets(tag)) with LazyLogging {
     }
   }
 
+  /** Get specific scenes from a bucket. Returns any scenes from the list of
+    * specified ids that are associated to the given bucket.
+    *
+    * @param bucketId UUID primary key of the bucket to limit results to
+    * @param sceneIds Seq[UUID] primary key of scenes to retrieve
+    */
+  def getBucketScenes(bucketId: UUID, sceneIds: Seq[UUID])
```

Comment:
Again from [here](https://github.com/azavea/raster-foundry/pull/564) this should be called `listBucketScenes` ?